### ISSUE TITLE
Multiple semantic nodes are not correctly linked during import

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#6235]
**Describe the bug**

Data structures with multiple semantic nodes at the same level in a single branch are no longer imported via JSON-LD.  Only one of the nodes is saved.  No errors or warnings are raised during the processing.

The parser correctly recognizes all of the data, and assigns them into branches. But when it is handed off to either resource.save() or to Resource.bulk_create(), the tiles are not persisted.

The model and incoming data are correct, as the structure can be created by hand and persisted, then exported. The data structure generated by the parser seems to be correct.

An example data structure that demonstrates this issue:

![Screen Shot 2020-06-01 at 2 28 24 PM](https://user-images.githubusercontent.com/871868/83456270-3bee5e00-a414-11ea-84a7-667acae145af.png)

The model and a sample record are attached: [test_case.zip](https://github.com/archesproject/arches/files/4713545/test_case.zip)

**To Reproduce**

Install the model. Try to import the record. View the record and note that only one of the branches is present.

To load the model with the command provided:

`python manage.py load_jsonld -s data -m negative`

**Expected behavior**
All the submitted data should be persisted, or there should be an error raised.

**Comment by azaroth42** [comment]
Further investigation reveals that the tile data is being persisted, but not retrieved.

Spitting out resource.tiles that is being saved, reveals the UUIDs of the tiles. Then using `whatisthis` reveals the data about the tile:

```
TileModel object (ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3)
_state
  <django.db.models.base.ModelState object at 0x11119fc10>
data
  {'54dcd606-a056-11ea-a45c-3af9d3b32b71': 'test place', '93a1e322-a056-11ea-a45c-3af9d3b32b71': '12,34'}
nodegroup_id
  4596224c-a056-11ea-a45c-3af9d3b32b71
parenttile_id
  aab89bfd-21ea-4336-b070-a602243f2b84
provisionaledits
  None
resourceinstance_id
  585d3961-5208-442d-96c1-f62858efaab8
sortorder
  0
tileid
  ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3
--------------------------------------------------------------------------------
Tile object (ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3)
_state
  <django.db.models.base.ModelState object at 0x1111bee50>
data
  {'54dcd606-a056-11ea-a45c-3af9d3b32b71': 'test place', '93a1e322-a056-11ea-a45c-3af9d3b32b71': '12,34'}
nodegroup_id
  4596224c-a056-11ea-a45c-3af9d3b32b71
parenttile_id
  aab89bfd-21ea-4336-b070-a602243f2b84
provisionaledits
  None
resourceinstance_id
  585d3961-5208-442d-96c1-f62858efaab8
sortorder
  0
tileid
  ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3
tiles
  []
```

(Changing the title of the issue to reflect that data is persisted, but not retrieved by either the json-ld export or the report view)

**Comment by azaroth42** [comment]
The tile data that gets persisted, as a CSV export from postgres:
[tile-data.csv.zip](https://github.com/archesproject/arches/files/4713671/tile-data.csv.zip)

**Comment by azaroth42** [comment]
The wrong parent tile data is being stored.

When I look at the resulting graph as turtle, it is disconnected because there are multiple Production nodes being generated. 

```
<http://localhost:8000/resources/585d3961-5208-442d-96c1-f62858efaab8> a ns1:E22_Human-Made_Object ;
    rdfs:label "Test Instance" ;
    ns1:P108i_was_produced_by <http://localhost:8000/tile/5505a18f-066a-4ad7-9797-45a6da82f723/node/7a49d3a4-a055-11ea-a45c-3af9d3b32b71> .

<http://localhost:8000/tile/5505a18f-066a-4ad7-9797-45a6da82f723/node/7a49d3a4-a055-11ea-a45c-3af9d3b32b71> a ns1:E12_Production ;
    rdfs:label "Test Production" ;
    ns1:P4_has_time-span <http://localhost:8000/tile/7573c4de-3583-4862-b285-f90d27093c7d/node/9ea5025a-a055-11ea-a45c-3af9d3b32b71> .

<http://localhost:8000/tile/7573c4de-3583-4862-b285-f90d27093c7d/node/9ea5025a-a055-11ea-a45c-3af9d3b32b71> a ns1:E52_Time-Span ;
    ns1:P1_is_identified_by <http://localhost:8000/tile/7573c4de-3583-4862-b285-f90d27093c7d/node/bb683240-a055-11ea-a45c-3af9d3b32b71> ;
    ns1:P82a_begin_of_the_begin "2020-06-01"^^xsd:dateTime ;
    ns1:P82b_end_of_the_end "2020-06-02"^^xsd:dateTime .

<http://localhost:8000/tile/7573c4de-3583-4862-b285-f90d27093c7d/node/bb683240-a055-11ea-a45c-3af9d3b32b71> a ns1:E33_E41_Linguistic_Appellation ;
    ns1:P190_has_symbolic_content "today" .

// Okay up to here...

/// But this should the Production event above
<http://localhost:8000/tile/7573c4de-3583-4862-b285-f90d27093c7d/node/7a49d3a4-a055-11ea-a45c-3af9d3b32b71> a ns1:E12_Production ;
    ns1:P67i_is_referred_to_by <http://localhost:8000/tile/aab89bfd-21ea-4336-b070-a602243f2b84/node/d6ff9160-a055-11ea-a45c-3af9d3b32b71> .

<http://localhost:8000/tile/aab89bfd-21ea-4336-b070-a602243f2b84/node/d6ff9160-a055-11ea-a45c-3af9d3b32b71> a ns1:E73_Information_Object ;
    rdfs:label "12 degrees" ;
    ns1:P43_has_dimension <http://localhost:8000/tile/29d7bf56-6c2e-4882-9a33-78b7f10ccdbf/node/fec9d764-a055-11ea-a45c-3af9d3b32b71> .

<http://localhost:8000/tile/29d7bf56-6c2e-4882-9a33-78b7f10ccdbf/node/fec9d764-a055-11ea-a45c-3af9d3b32b71> a ns1:E54_Dimension ;
    ns1:P90_has_value 12 .


// And this should also just be the first Production event

<http://localhost:8000/tile/aab89bfd-21ea-4336-b070-a602243f2b84/node/7a49d3a4-a055-11ea-a45c-3af9d3b32b71> a ns1:E12_Production ;
    ns1:P7_took_place_at <http://localhost:8000/tile/ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3/node/4596224c-a056-11ea-a45c-3af9d3b32b71> .

<http://localhost:8000/tile/ee5eb9b4-81ad-4b73-ade7-10ea511c6ab3/node/4596224c-a056-11ea-a45c-3af9d3b32b71> a ns1:E53_Place ;
    rdfs:label "test place" ;
    ns1:P168_place_is_defined_by "12,34" .
```

I know that we had this in the past, but it was solved and has now reverted?

**Comment by chiatt** [comment]
@azaroth42 is this still a problem?

**Comment by azaroth42** [comment]
Yes it is still not fixed in master. The fix is in #6237.

**Comment by apeters** [comment]
@azaroth42 this was fixed by #6237
